src/components/utils/string.js

```diff
@@ -11,7 +11,7 @@ const unCamel = str => str.replace(camelCaseRe, ' $1').split(' ')
 
 const compactFlat = compose(cleanFalsy, flatten)
 
-const wordSplitRe = /[^\s_\-]+/g
+const wordSplitRe = /[^\s_-]+/g
```

Comment:
Why this change?

Reply:
linter complained about unnecessary escapes when I upgraded it.